epub33/core/index.html

```diff
@@ -5746,11 +5746,12 @@ <h4>File Paths and File Names</h4>
 				<section id="sec-file-names-to-path-names">
 					<h4>Deriving File Paths of Files</h4>
 
-					<p>To derive the <a>File Path</a> of a file or directory <var>file</var> in the <a
-							href="#sec-container-abstract">OCF Abstract Container</a> apply the following steps
+					<p>To derive the <a>File Path</a>, given a file or directory <var>file</var> in the <a
```

Comment:
I believe the convention is to put dfn tags around the label to be able to reference the algorithm and get the list of references to it (even if it seems to not be referenced in this document right now):

`<dfn id="derive-the-file-path">derive the <a>File Path</dfn>`

The link to the definition of File Path makes this a bit weird, though. It works, but depending on which half of the definition you click you either get the references list or go the terminology.

You could maybe assume readers will know its a terminology link from the capital letters and omit the link.

Reply:
Hm. I may not understand what you say, but... the convention is, as far as I could see, the the `<dfn>` is in the terminology section, linking to the section that contains a more detailed definition (if applicable). This follows the same pattern...

Reply:
> the the `<dfn>` is in the terminology section

Right, I mean the convention for defining the algorithm.

If you go to the URL standard, etc., or even to pub-manifest, the name of the algorithm is defined in `dfn` tags. This lets you reference the algorithm by name, so you can later write stuff like:

"In this case, you need to `<a>`derive the File Path`</a>` of ...".

Here's an example for the "normalize data" algorithm in pub-manifest: https://www.w3.org/TR/pub-manifest/#normalize-data

If you click on the permalink it will show you where else in the document the algorithm is referenced.

Reply:
I understand that. My problem is whether this (sound!) convention is not something we should use elsewhere in the document, too.

What I would do is to follow this style for this section (I prefer not to add the link because it would make things confusing) and we may want to sweep through the text at some point to see if there are other places to update accordingly.

Reply:
(Done in the latest commit)

Reply:
Right, I hate links within links, so all good now from my perspective.